src/main/java/calculator/operator/PositiveNumber.java

```diff
@@ -0,0 +1,54 @@
+package calculator.operator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class PositiveNumber {
+
+    private int value;
+
+    public PositiveNumber(int value) {
+        validateRange(value);
+        this.value = value;
+    }
+
+    public PositiveNumber sum(PositiveNumber number) {
+        this.value += number.value;
+        return this;
+    }
+
```

Comment:
```suggestion

    public PositiveNumber sum(PositiveNumber number) {
        return new PositiveNumber(this.value + number.value);
    }

```
새로운 객체를 만들어서 리턴하는 것은 어떨까요? equals & hashCode 를 정의해 주셔서 내부의 value가 동일하면 동일한 객체로 취급하고 있습니다. 

```java
PositiveNumber one = new PositiveNumber(1);
PositiveNumber two = new PositiveNumber(2);

PositiveNumber three = one.sum(two);
```

이 경우 one의 value 값은 몇인 게 자연스러울까요?? 
메서드명도 plus가 좀 더 어울릴 것 같기는 하네요!